LayoutTests/imported/w3c/web-platform-tests/css/cssom/shorthand-values-expected.txt

```diff
@@ -14,7 +14,7 @@ PASS The serialization of overflow-x: scroll; overflow-y: hidden; should be cano
 PASS The serialization of overflow-x: scroll; overflow-y: scroll; should be canonical.
 PASS The serialization of outline-width: 2px; outline-style: dotted; outline-color: blue; should be canonical.
 PASS The serialization of margin-top: 1px; margin-right: 2px; margin-bottom: 3px; margin-left: 4px; should be canonical.
-PASS The serialization of list-style-type: circle; list-style-position: inside; list-style-image: initial; should be canonical.
+FAIL The serialization of list-style-type: circle; list-style-position: inside; list-style-image: initial; should be canonical. assert_equals: expected "list-style: inside circle;" but got "list-style-type: circle; list-style-position: inside; list-style-image: initial;"
```

Comment:
I plan to do that in another change later; I have it stashed elsewhere. The reason I didn’t do it now is that we won’t pass the revised test because we don’t yet support initial values properly.

Reply:
Another option is that I revise the test so that it still tries to test with `initial` and just has an appropriate expectation.

Reply:
Either is fine by me, I just don't want this to slip through the cracks.

Reply:
I will follow through; this will not slip through the cracks.